install

```diff
@@ -179,6 +179,16 @@ install_desktop_environment() {
       && success "urxvt and xterm installed"
 }
 
+install_vimplug() {
```

Comment:
What's the advantage of downloading directly over using a submodule, I know submodules are a pain, but the version get's locked so it's a safeguard if something may be incompatible with later versions